libraries/AC_Avoidance/AP_OADatabase.cpp

```diff
@@ -102,6 +110,13 @@ const AP_Param::GroupInfo AP_OADatabase::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO_FRAME("ALT_MIN", 8, AP_OADatabase, _min_alt, 0.0f, AP_PARAM_FRAME_COPTER | AP_PARAM_FRAME_HELI | AP_PARAM_FRAME_TRICOPTER),
 
+    // @Param: EN_TRACKED
+    // @DisplayName: OADatabase enable Tracked Objects
+    // @Description: Enable storing tracked objects in the OAdatabase. This is only possible via mavlink message OBSTACLE_DISTANCE_3D
+    // @Values: 0:Disabled,1:Enabled
+    // @User: Advanced
+    AP_GROUPINFO("EN_TRACKED", 9, AP_OADatabase, _tracked_obs_enable, 0),
```

Comment:
if we need the enable how about we instead change it to a count of the number of dynamic obstacles tracked.  by the way, I'm not sure that "tracked" is going to help users understand the difference between these dynamic obstacles vs the fence points so maybe we should come up with more clear wording... I think we should call them "dynamic objects"

Reply:
.. and I think we should try and reduce the use of synonyms.  So I thinkwe may be using "obstacle", "object" and "item" interchangeably.  Let's try and stick to just one word.. I'm not saying let's go back and rename everything but at least for the new code, where possible, let's try and reduce the number of synonyms used.